package.json

```diff
@@ -25,6 +25,7 @@
     "compression": "^1.1.0",
     "connect": "^3.2.0",
     "connect-static-file": "^1.1.2",
+    "log4js": "^1.1.0",
```

Comment:
Go ahead and lock the dependency here to the latest current version. (I locked the dependencies recently to avoid any unknown breaking changes)